### PR TITLE
MAT-6395: Fix inconsistency from measureState

### PR DESCRIPTION
**Pull request by mcmcphillips**
## MADiE PR

Jira Ticket: [MAT-6395](https://jira.cms.gov/browse/MAT-6395)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included
* [ ] No extemporaneous files are included (i.e Complied files or testing results)
* [ ] This PR is into the **correct branch**.
* [ ] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

File: src/components/testCaseConfiguration/expansion/Expansion.tsx [hunk 1]
```diff
@@ -110,7 +110,7 @@ const Expansion = () => {
       .then(() => {
         handleToast("success", "Expansion details Updated Successfully", true);
         // To disable Run test case button, until cqmMeasure is rebuilt
-        setExecutionContextReady(false);
+        // setExecutionContextReady(false);
```
Comment (jkotanchik-SB) [inline src/components/testCaseConfiguration/expansion/Expansion.tsx]: If we need to keep this code around (for A/B testing or something), please comment stating why. If not, please remove with it's accompanying code comment.

File: src/components/testCaseLanding/qdm/TestCaseList.tsx [hunk 2]
```diff
@@ -176,13 +176,7 @@ const TestCaseList = (props: TestCaseListProps) => {
...
     }
   }, [measure]);
-
-  useEffect(() => {
-    if (testCases?.length != measure?.testCases?.length) {
-      const newMeasure = { ...measure, testCases };
-      updateMeasure(newMeasure);
-    }
-  }, [testCases]);
+  // removed area where test cases were saved to measure here with updateMeasure. Unsure why this existed. Doesn't break tests.
```
Comment (jkotanchik-SB) [inline src/components/testCaseLanding/qdm/TestCaseList.tsx]: This should be a commit message.

**Review by jkotanchik-SB — APPROVED** [review]
